test_data/rules/avoid_function_literals_in_foreach_calls.dart

```diff
@@ -9,12 +9,14 @@ class Person {
 }
 
 void main() {
-  Iterable<String> people;
+  Iterable<String> people = [];
```

Comment:
Should we add something to the test framework to fail a test if there are any diagnostics other than the lints being tested for? Seems like this is something that should have been caught when switching to analyze test code in null safety by default.

Reply:
Good question. We're currently ignoring diagnostics and I suspect there are a ton.

https://github.com/dart-lang/linter/issues/2760